benchmarks/retrieval/retrieve.py

```diff
@@ -84,7 +100,7 @@ def main():
     print("Calculating metrics...")
     results = calc_aggregate([Rprec, P @ 1, R @ 3, nDCG @ 3, MAP, MRR], golden_docs, retrieved_docs)
     results = {str(key): value for key, value in results.items()}
-
+    print(results)
```

Comment:
If you sync to master, you'll see you won't need this print statement.
The problem was lines 118-120 were indented too much to the right.